helm/g2p-sandbox-fynarfin-demo/values.yaml

```diff
@@ -0,0 +1,482 @@
+ph-ee-g2psandbox:
```

Comment:
add just the  values you are changing compared to your base chart.(g2p Sandbox)

Reply:
Cleanup activity can be addressed later.